.travis.yml

```diff
@@ -1,9 +1,24 @@
 language: ruby
 rvm:
+- 2.2
 - 2.1
-- 2.0.0
-- 1.9.3
+- 2.0
 before_script: bundle update
 script: "script/cibuild"
 sudo: false
 cache: bundler
+
+matrix:
+  include:
+    - # GitHub Pages
+      rvm: 2.1.1
+      env: GH_PAGES=true
```

Comment:
In the Gemfile on [line 4](https://github.com/jekyll/jekyll-sitemap/pull/83/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478R4), this will trigger a build with the whole `github-pages` gem.